src/test/rustdoc-ui/intra-doc/html-as-generics-intra-doc.rs

```diff
@@ -0,0 +1,25 @@
+#![deny(rustdoc::invalid_html_tags)]
+#![deny(rustdoc::broken_intra_doc_links)]
+
+pub struct ExistentStruct<T>(T);
+
+/// This [test][ExistentStruct<i32>] thing!
+pub struct NoError;
+
+/// This [ExistentStruct<i32>] thing!
+//~^ERROR unclosed HTML tag `i32`
```

Comment:
Please put space between `^` and `ERROR` (took me a while to get my head around it :laughing: ).